src/Language/Haskell/LSP/Client.hs

```diff
@@ -36,11 +36,13 @@
 --
 --   * Implement proper exception handling.
 --
-module LSP.Client
+module Language.Haskell.LSP.Client
```

Comment:
I deliberately didn't put `Language.Haskell` there, because this client library doesn't have anything to do with the Haskell language in particular, except that it is written in Haskell of course.

Reply:
Woops, didn't mean to include this, was using it in my own package

Reply:
I plan on using this package with a testing framework for `haskell-ide-engine` though, so if you would consider making it a part of this ecosystem then the name might fit better